cert/selfsigned.go

```diff
@@ -100,6 +100,52 @@ func dnsNames(tlsExtraDomains []string) (string, []string) {
 	return host, dnsNames
 }
 
+// Outdated returns whether the given certificate is outdated wrt the IPs and
```

Comment:
Naming suggestion: `IsOutdated`?

Reply:
wrt -> w.r.t

Reply:
`godoc` comment could also use further elaboration on what "outdated" means in this context (IPs or domains aren't a sub-set of what's currently in the cert), as a few of our related packages use this `cert` package to handle routine cert operations.

Reply:
Done